components/chainSelect/ChainSelect.tsx

```diff
@@ -175,13 +171,26 @@ const ChainSelect = () => {
     return "";
   };
 
-  const getNodeFromArray = (nodeArray: { address: string; provider: string }[]) => {
+  const getNodeFromArray = async (nodeArray: { address: string; provider: string }[]) => {
     // only return https connections
-    const secureNodes = nodeArray.filter((node) => node.address.includes("https://"));
+    const secureNodes = nodeArray
+      .filter((node) => node.address.includes("https://"))
+      .map(({ address }) => address);
+
     if (secureNodes.length === 0) {
       throw new Error("No SSL enabled RPC nodes available for this chain");
     }
-    return secureNodes[0].address;
+
+    try {
+      for (const node of secureNodes) {
+        // test client connection
+        const client = await StargateClient.connect(node);
+        await client.getHeight();
+        return node;
+      }
+    } catch {}
```

Comment:
Here you need to try/catch inside of the loop. Otherwise the first exception will be handled here and lead to "No RPC nodes available for this chain"